pyiron_base/project/generic.py

```diff
@@ -367,6 +367,52 @@ def create_job_class(
             executable_str=executable_str,
         )
 
+    def create_job_step(
+        self,
+        job_name,
+        executable_str,
+        write_input_funct=None,
+        collect_output_funct=None,
+        input_dict=None,
+        conda_environment_path=None,
+        conda_environment_name=None,
+        input_file_lst=None,
+        execute_job=True,
+    ):
+        """
+
```

Comment:
Add example in documentation and explanation of the function as well as a warning that cloud pickle is used for storage.